test/kitchen/test/integration/win-agent-with-cws-option/rspec_datadog/win-agent-with-cws-option_spec.rb

```diff
@@ -0,0 +1,131 @@
+require 'spec_helper'
+require 'windows_npm_spec_helper' # for is_windows_service_disabled
+
+
+shared_examples_for 'a Windows Agent with CWS driver disabled' do
+    # We retrieve the value defined in kitchen.yml because there is no simple way
+    # to set env variables on the target machine or via parameters in Kitchen/Busser
+    # See https://github.com/test-kitchen/test-kitchen/issues/662 for reference
+    let(:expect_cws_installed) {
+        parse_dna().fetch('dd-agent-rspec').fetch('cws_included')
+    }
+
+    it 'has the service disabled' do
+        
+        if expect_cws_installed
+            expect(is_windows_service_disabled("ddprocmon")).to be_truthy
+        end 
+    end
+end
+  
+shared_examples_for 'a Windows Agent with CWS driver installed' do
+    it 'has system probe service installed' do
+        expect(is_windows_service_installed("datadog-system-probe")).to be_truthy
+    end
+
+    # We retrieve the value defined in kitchen.yml because there is no simple way
+    # to set env variables on the target machine or via parameters in Kitchen/Busser
+    # See https://github.com/test-kitchen/test-kitchen/issues/662 for reference
+
+    # the `let` has to be inside an it block.
+    let(:expect_cws_installed) {
+        parse_dna().fetch('dd-agent-rspec').fetch('cws_included')
+    }
+
+    it 'has properly installed driver files' do
+        if expect_cws_installed
+            expect(is_windows_service_installed("datadog-security-agent")).to be_truthy
+            expect(is_windows_service_installed("ddprocmon")).to be_truthy
+
+            program_files = safe_program_files
+            expect(File).to exist("#{program_files}\\DataDog\\Datadog Agent\\bin\\agent\\ddprocmon.cat")
+            expect(File).to exist("#{program_files}\\DataDog\\Datadog Agent\\bin\\agent\\ddprocmon.sys")
+            expect(File).to exist("#{program_files}\\DataDog\\Datadog Agent\\bin\\agent\\ddprocmon.inf")
```

Comment:
Are these paths correct? If so, why are the `ddprocmon.*` files in `bin\agent` and not in `bin\agent\driver` with the rest of the drivers?

Reply:
they're currently correct, as the procmon merge module isn't putting them in the right place.
Need to fix the procmon merge module, and then would (in a separate pr) update paths to match